src/freenet/client/filter/FilterCallback.java

```diff
@@ -26,18 +26,28 @@
 	 */
 	public String processURI(String uri, String overrideType, boolean noRelative, boolean inline) throws CommentException;
 
+	/**
```

Comment:
Adding methods to public interfaces is dangerous… I didn’t find a match in anything I have checked out but we can’t really know what we’ll break with this as long as the interface is public. 😕

Reply:
Here I think I actually need it.